tools/cold-store/src/cli.rs

```diff
@@ -262,3 +282,139 @@ fn _get_with_fallback<T: near_primitives::borsh::BorshDeserialize>(
             .unwrap_or_else(|| panic!("No value for {} {:?} in any storage", col, key)),
     }
 }
+
+#[derive(clap::Parser)]
+struct PrepareHotCmd {
+    /// The relative path to the rpc store that will be converted to a hot store.
+    /// The path should be relative to the home_dir e.g. hot_data
+    #[clap(short, long)]
+    store_relative_path: String,
+}
+
+impl PrepareHotCmd {
+    pub fn run(
+        &self,
+        storage: &NodeStorage,
+        home_dir: &Path,
+        near_config: &NearConfig,
+    ) -> anyhow::Result<()> {
+        let _span = tracing::info_span!(target: "prepare-hot", "run");
+
+        let path = Path::new(&self.store_relative_path);
+        tracing::info!(
+            target = "prepare-hot",
```

Comment:
`target = ` has a completely different meaning. It'll log the value as that key. `target: ` lets you control logging via `RUST_LOG`.

```suggestion
            target: "prepare-hot",
```